tensorrt_llm/_torch/model_config.py

```
@@ -334,12 +356,14 @@ def load_modelopt_quant_config(quant_config_file, checkpoint_dir,
             except Exception:
                 logger.info(
-                    f"No quant_cfg.json found for layer quant info, using hf_quant_config.json."
+                    "No quant_cfg.json found for layer quant info, using hf_quant_config.json."
                 )
             json_quant_configs.update(json_extended_quant_configs)
             # kv_cache_quant_algo is global regardless of MIXED_PRECISION
-            kv_cache_quant_algo = json_quant_configs.get(
-                'kv_cache_quant_algo', None)
+            kv_cache_quant_algo = (QuantAlgo(
+                json_quant_configs['kv_cache_quant_algo']) if
+                                   json_quant_configs.get('kv_cache_quant_algo')
```

Flake8 (7.3.0)

[error] continuation line over-indented for hanging indent

(E126)